config/Version.public.xcconfig

```diff
@@ -1,9 +1,9 @@
-VERSION_SHORT=2.1
+VERSION_SHORT=2.2
 
 // Public long version. Example: 2.0.0.0. The last 0 means the first build for
 // version 2.0.0
-VERSION_LONG=2.1.0.0
+VERSION_LONG=2.2.0.0
 
 // This is the value for the CFBundleVersion it should be incremented on every
 // build that gets distributed
-BUILD_NUMBER=11204
+BUILD_NUMBER=11205
```

Comment:
This was made by the logic from the release toolkit plugin, thanks again @loremattei.